polygeist/lib/Conversion/PolygeistToLLVM/PolygeistToLLVM.cpp

```diff
@@ -1548,6 +1569,11 @@ struct ConvertPolygeistToLLVMPass
 
       patterns.add<URLLVMOpLowering>(converter);
 
+      // cgeist already introduces these globals, so we can drop the ones coming
```

Comment:
Does cgeist always introduces all spirv globals? do we need to check/assert if it already exists in the module before erasing it?

Reply:
It does, as they're part of a header included by the runtime. I will add checks anyway

Reply:
Added the checks.